initializr-be/src/main/java/io/axoniq/initializr/extension/dependency/JobRunrProjectCustomizer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021. AxonIQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.axoniq.initializr.extension.dependency;
+
+import io.spring.initializr.generator.buildsystem.Build;
+import io.spring.initializr.generator.buildsystem.Dependency;
+import io.spring.initializr.generator.project.ProjectDescription;
+import io.spring.initializr.generator.project.contributor.ProjectContributor;
+import io.spring.initializr.generator.spring.build.BuildCustomizer;
+import io.spring.initializr.generator.version.Version;
+import io.spring.initializr.generator.version.VersionReference;
+
+/**
+ * A {@link ProjectContributor} that adds the correct JobRunr starter, based on the boot version.
+ *
+ * @author Gerard Klijs
+ */
+public class JobRunrProjectCustomizer implements BuildCustomizer<Build> {
+
+    private final Version springBootVersion;
+
+    JobRunrProjectCustomizer(ProjectDescription projectDescription){
+        springBootVersion = projectDescription.getPlatformVersion();
+    }
+
+    @Override
+    public void customize(Build build) {
+        if (springBootVersion.getMajor() == 3) {
+            build.dependencies().add(
+                    "jobrunr-starter",
+                    Dependency
+                            .withCoordinates("org.jobrunr", "jobrunr-spring-boot-3-starter")
+                            .version(VersionReference.ofValue("6.0.0"))
+            );
+        }else{
```

Comment:
Super nit: indentation is slightly off.
```suggestion
        } else {
```

Reply:
Reformatted the code.